supervisely/app/widgets/custom_models_selector/custom_models_selector.py

```diff
@@ -385,19 +394,35 @@ def _generate_table_rows(self, checkpoint_infos: List[CheckpointInfo]) -> List[D
                     checkpoint=checkpoint_info,
                     task_type=checkpoint_info.task_type,
                 )
-                table_rows.append(model_row)
+                table_rows[checkpoint_info.task_type].append(model_row)
             except:
                 continue
+        table_rows = dict(table_rows)
```

Comment:
- check `disable_table` and `enable_table` methods (`self._rows` is `dict` now, checkpoint_selector attribute of `dict` key?). Lines 361, 366
- typing `list` > `dict`. Line 334